### PR TITLE
Add Sound.Play overloads that play a random sound from a list

### PR DESCRIPTION
**Pull request by abcdefg30**
Following a discussion on IRC.

### CODE REVIEW

File: OpenRA.Game/Sound/Sound.cs [hunk 1]
```diff
@@ -147,6 +147,16 @@ public void UnmuteAudio()
 		public ISound PlayLooped(SoundType type, string name) { return Play(type, null, name, true, WPos.Zero, 1f, true); }
 		public ISound PlayLooped(SoundType type, string name, WPos pos) { return Play(type, null, name, false, pos, 1f, true); }
 
+		public ISound Play(SoundType type, string[] names, Player player = null, float volumeModifier = 1f)
+		{
+			return Play(type, player, names.Random(Game.CosmeticRandom), false, WPos.Zero, volumeModifier);
```
Comment (pchote) [inline OpenRA.Game/Sound/Sound.cs]: We have been aiming to migrate `Game.CosmeticRandom` to `world.LocalRandom`. Can we add a `World` parameter and use that here?

**Comment by abcdefg30** [comment]
Updated.

**Review by pchote — DISMISSED** [review]

**Comment by reaperrr** [comment]
This breaks at least weapon firing sounds.

**Comment by abcdefg30** [comment]
Ah I see what you mean, they were more quiet than usual because I passed the wrong bool for distance calculation. Should be fixed.

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/937b4e1696fdb7a67837dcfad9704810a7dfcfaa...8c63a73c3d1e5034d72657d3c68b64a544fe090e

**Pull request merged**